### PR TITLE
update api error message to a user friendly error message

### PR DESCRIPTION
**Pull request by pradeepdewda-okta**
## Description:
Update api error message

1. This fixes the subset of users of Broadcom hitting "Invalid token provided" error from API
2. Clear the state token from cookie store, if the token is no more valid.

For more history about the bug checkout
https://oktainc.atlassian.net/browse/OKTA-374956
https://oktainc.atlassian.net/browse/OKTA-367327

## PR Checklist

- [ ] Have you verified the basic functionality for this change?

- [ ] Added unit tests?

- [ ] Added e2e tests

- [ ] Did you follow our [Security Best Practices](https://oktawiki.atlassian.net/wiki/display/eng/Security+Best+practices)?

### Screenshot/Video:
Before
Screen Shot 2021-03-03 at 8 15 35 PM
After
Screen Shot 2021-03-08 at 4 42 19 PM

### Issue:

- [OKTA-376620](https://oktainc.atlassian.net/browse/OKTA-376620)

### CODE REVIEW

File: packages/@okta/i18n/src/properties/login.properties [hunk 1]
```diff
@@ -69,7 +69,7 @@ errors.E0000007 = Not found: {0}
 errors.E0000008 = The requested path was not found
 errors.E0000009 = Internal Server Error
 errors.E0000010 = Service is in read only mode
-errors.E0000011 = Invalid token provided
+errors.E0000011 = Your session has expired, Please refresh the page to sign in.
```
Comment (nikhilvenkatraman-okta) [inline packages/@okta/i18n/src/properties/login.properties:72]: Can you update the tests ? 
https://github.com/okta/okta-signin-widget/blob/master/test/unit/helpers/xhr/ERROR_invalid_token.js#L6
https://github.com/okta/okta-signin-widget/blob/master/test/unit/spec/LoginRouter_spec.js#L849

Comment (jmelberg-okta) [reply packages/@okta/i18n/src/properties/login.properties:72]: Given the high visibility of this error message, let's have someone from doc 👍🏼  this new message.

```suggestion
errors.E0000011 = Your session has expired. Please refresh the page to sign in.
```

File: src/util/RouterUtil.js [hunk 2]
```diff
@@ -106,6 +107,8 @@ fn.routeAfterAuthStatusChangeError = function (router, err) {
 
   // Token has expired - no longer valid. Navigate back to primary auth.
   if (err.errorCode === ErrorCodes.INVALID_TOKEN_EXCEPTION) {
+    // If token is invalid then it must be deleted from cookie
+    CookieUtil.removeOktaStateToken();
```
Comment (jmelberg-okta) [inline src/util/RouterUtil.js:111]: This looks like it [happens by default](https://github.com/okta/okta-auth-js/blob/236a4a4e54e634a36e004be93a4cda2580198dd2/test/spec/http.js#L263-L276) in AuthJS when we encounter this error code, is that not happening?

cc: @aarongranick-okta

File: packages/@okta/i18n/src/properties/login.properties (hunk 1 above)
Comment (pradeepdewda-okta) [reply packages/@okta/i18n/src/properties/login.properties:72]: @grahamsmith-okta can you review the messaging around this ?

Comment (grahamsmith-okta) [reply packages/@okta/i18n/src/properties/login.properties:72]: I would drop "has" and "please" - Your session expired. Refresh the page to sign in.
Also, in E000010 hyphenate read-only - Service is in read-only mode

Comment (pradeepdewda-okta) [reply packages/@okta/i18n/src/properties/login.properties:72]: addressed comments here on 5.4 base : https://github.com/okta/okta-signin-widget/pull/1733

**Comment by pradeepdewda-okta** [comment]
New PR on 5.4 : https://github.com/okta/okta-signin-widget/pull/1733

**Pull request closed without merging**